lib/cluster.py

```diff
@@ -120,6 +122,88 @@ def reset(self, config):
         )
         assert(status == 0)
 
+        # Validate CBGT
+        if not self.validate_cbgt_pindex_distribution_retry():
+            self.save_cbgt_diagnostics()
+            raise Exception("Failed to validate CBGT Pindex distribution")
+
+        print(">>> Detected valid CBGT Pindex distribution")
```

Comment:
@ashvindersingh the rest of the file is using `print`, so I will leave it as-is.  If we want to mass-convert the file to using the logging module, it should be in a separate PR.